test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -2809,27 +2923,28 @@ public void testAuthzAuthorityAcl_noResults() throws Exception {
   @Test
   public void testAuthzAuthorityAcl_permit() throws Exception {
     executeUpdate("create table acl(id integer, gsa_permit_users varchar(20))");
+    executeUpdate("create table data(id integer, author varchar(20))");
```

Comment:
This shouldn't be here at all, but elsewhere the pattern is "create data, insert data, create acl, insert acl".

Reply:
Done.